src/MessagePack.UnityClient/Assets/Scripts/MessagePack/Unity/MessagePackWindow.cs

```diff
@@ -150,7 +150,7 @@ async void OnGUI()
                 invokingMpc = true;
                 try
                 {
-                    var log = await ProcessHelper.InvokeProcessStartAsync("dotnet", "mpc " + commnadLineArguments);
+                    var log = await ProcessHelper.InvokeProcessStartAsync("mpc " + commnadLineArguments);
```

Comment:
I suspect the compile break problem here is you need to keep two parameter. Maybe split: `"mpc", commnadLineArguments`

Also maybe fix the typo in `commnadLineArguments`.

Reply:
Ah sorry, mistake with #1109, I should remove this commit in this PR.